### ISSUE TITLE
Right hand rule for rotational quantities

### ISSUE THREAD

**Issue by owoodman** [OSTP/PhysicsArtofModelling#179]
I was thinking we could call the right hand rule for rotational quantities "the right hand rule for axial vectors" because I think it's a bit more intuitive than "the right hand rule for pseudovectors", and it looks like the two are synonymous .

**Comment by ryanmartinneutrino** [comment]
Yes, I think this would be good, then we have RHR for axial vectors and RHR for cross-product. We'll have to add something about this in the Vector appendix. I think we can define axial vector as the "type of vector that is used to describe rotational quantities that specify an axis of rotation and a direction about that axis".  We'll need a footnote or something to define an axial vector correctly, based on the property under reflection - the example in wikipedia however requires a knowledge of current and B field, but maybe we can find a better example, perhaps about describing something to your mirror self...

**Comment by owoodman** [comment]
Ok. I wrote something brief in the vector appendix a couple days ago (screenshot attached) but I will update it so that it reflects this. I found this online: https://www.quora.com/What-is-a-pseudovector.  I think the example of the car wheels reflected in the mirror makes sense. [
![screen shot 2018-07-19 at 2 03 53 pm](https://user-images.githubusercontent.com/38730915/42961635-4b5e7676-8b5d-11e8-9131-306cddc915c7.png)
](url)
